public/views/app.html

```diff
@@ -57,7 +57,7 @@ <h6>{{game.errorMsg}}</h6>
           <h6>{{game.message}}</h6>
         </div>
         <div class="button-div">
-          <span><button class= "user-button" ng-click="sendGameInvite()">Invite</button></span>
+          <span><button class= "user-button" ng-click="sendGameInvite()" ng-if="userInfo.user.email">Invite</button></span>
```

Comment:
I'll say the same thing here, adjust your variable naming pattern. Make sure it conveys a description of the task.

Reply:
@Orlayhemmy Not particularly sure what variable you are referring to. Please can you be more specific with this feedback

Reply:
Sorry, my mistake I was referring to the "ng-if" directive. It's an angular property.